linters/remark-lint-no-br-in-tables.js

```diff
@@ -0,0 +1,92 @@
+import { visit } from 'unist-util-visit'
+
+const remarkLintNoBrInTables = (severity = 'warning') => {
```

Comment:
I believe this causes rendering issues. If so, it should be an error

```suggestion
const remarkLintNoBrInTables = (severity = 'error') => {
```

Reply:
i left the severity default to warning, but it'll be called error in the runLint.js because this way we can often switch to warning easier.